docs/user_guide/source/components/engine.rst

```diff
@@ -323,7 +325,7 @@ PerformsPuts
 
 
 PerformsDataWrite
```

Comment:
Not an issue with this update, but everywhere that says "Performs*" in headers here should be "Perform" (without the s).

Reply:
i fixed that too.